send/ldap_it4i

```diff
@@ -0,0 +1,164 @@
+#!/usr/bin/perl
+use strict;
+use warnings;
+no if $] >= 5.017011, warnings => 'experimental::smartmatch';
+use Net::LDAPS;
+use Net::LDAP::Entry;
+use Net::LDAP::Message;
+use Net::LDAP::LDIF;
+use ScriptLock;
+
+sub process_update;
+sub process_archive;
+
+# Constants and counters
+my $counter_add = 0;
+my $counter_update = 0;
+my $counter_archive = 0;
+my $counter_fail = 0;
+
+# Define service
+my $service_name = "ldap_it4i";
+
+# GEN folder location
+my $facility_name = $ARGV[0];
+chomp($facility_name);
+my $service_files_base_dir="../gen/spool";
+my $service_files_dir="$service_files_base_dir/$facility_name/$service_name";
+my $service_file = "$service_files_dir/$service_name.ldif";
+
+# BASE DN
+open my $file, '<', "$service_files_dir/baseDN";
+my $base_dn = <$file>;
+chomp($base_dn);
+close $file;
+
+# propagation destination
+my $namespace = $ARGV[1];
+chomp($namespace);
+
+# create service lock
+my $lock = ScriptLock->new($facility_name . "_" . $service_name . "_" . $namespace);
+($lock->lock() == 1) or die "Unable to get lock, service propagation was already running.";
+
+# init configuration
+my @conf = init_config($namespace);
+my $ldap_location = resolve_pdc($conf[0]);
```

Comment:
We can't use function to resolve PDC (primary domain controller) from DNS, since the LDAP instance is not an AD. We should use raw value from the config (`$conf[0]`). Please make note about it in the script itself.

Reply:
Added.